samples/Sentry.Samples.Google.Cloud.Functions/Function.cs

```diff
@@ -16,6 +16,7 @@ public class Function : IHttpFunction
 
     public Task HandleAsync(HttpContext context)
     {
+        _logger.LogTrace("LogLevel.Trace is not configured to be sent as structured log");
```

Comment:
```suggestion
        // Configure structured logging via appsettings.json (Logging:Sentry:LogLevel:)
        _logger.LogTrace("LogLevel.Trace is not configured to be sent as structured log");
```

Just in case people haven't seen the other samples where there are comments that make this clear.